src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/form.html.twig

```diff
@@ -31,6 +32,20 @@
 {% endblock %}
 
 {% block employee_form %}
+  {% if isRestrictedAccess %}
+    <div class="alert alert-info" role="alert">
+      <div class="alert-text">
+        {{ 'Note that this profile will be edited for all the stores with which it is associated.'|trans({}, 'Admin.Advparameters.Help') }}<br>
+      </div>
+    </div>
+  {% else %}
+    <div id="multistore-alert" class="alert alert-info" role="alert">
+      <div class="alert-text">
+        {{ 'Note that this feature is available in all shops context only. It will be added to all your stores.'|trans({}, 'Admin.Advparameters.Help') }}<br>
```

Comment:
```suggestion
        {{ 'Note that this feature is available in all shops context only. It will be added to all your stores.'|trans({}, 'Admin.Notification.Info') }}<br>
```
If this is used elsewhere in the software (and I have noticed on other PRs that it is) it should be in a more generic domain. :-)